dvc/remote/local/__init__.py

```diff
@@ -360,37 +365,23 @@ def status(
         logger.info(
             "Preparing to collect status from {}".format(remote.path_info)
         )
-        title = "Collecting information"
-
         ret = {}
-
-        progress.set_n_total(1)
-        progress.update_target(title, 0, 100)
-
-        progress.update_target(title, 10, 100)
-
         ret = self._group(checksum_infos, show_checksums=show_checksums)
         md5s = list(ret)
 
-        progress.update_target(title, 30, 100)
-
+        logger.info("Collecting information for local cache...")
```

Comment:
How about we use this message (or maybe something like `Collecting info for s3://mybucket/path`) in the progress bars themselves?

Reply:
@efiop , instead of showing the checksums?